content/javascript/concepts/data-types/data-types.md

```diff
@@ -14,20 +14,26 @@ CatalogContent:
-| `Boolean`   | This data type only has two possible values — either `true` or `false`.                                                                       |
-| `Null`      | This data type represents the intentional absence of a value, and is represented by the keyword `null`.                                       |
-| `Undefined` | This data type is denoted by the keyword `undefined`. It also represents the absence of a value though it has a different use than `null`.    |
-| `Symbol`    | A newer feature to the language, symbols are unique identifiers, useful in more complex coding. No need to worry about these for now.         |
-| `Object`    | Collections of related data.                                                                                                                  |
-
-The first 7 of those types are considered primitive data types. They are the most basic data types in the language. Objects are a little more complex.
+**Data types** are how data in programming is classified. In JavaScript, there are eight fundamental data types: 
+
+## Primitive Data Types 
+
+The table below holds the seven primitive data types. They are the most basic data types in the language. Objects are a little more complex.
+
+| Data Type | Wrapper Object   | Description                                                                                                                                   |
+|-----| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `number` | [`Number`](https://www.codecademy.com/resources/docs/javascript/number-methods)    | Any number, including numbers with decimals: `1`, `-2`, `99`, `3.14`.                                                                         |
+| `bigInt` | `BigInt`    | Any number, greater than 2^53-1 or less than -(2^53-1) with `n` appended to the number: `1234567890123456n`.                                  |
+| `string` | [`String`](https://www.codecademy.com/resources/docs/javascript/strings)    | Any grouping of characters on your keyboard (letters, numbers, spaces, symbols, etc.) surrounded by single `''` or double `""`. |
+| `boolean` | `Boolean`   | This data type only has two possible values — either `true` or `false`.                                                                       |
+| `null` | `Null`      | This data type represents the intentional absence of a value, and is represented by the keyword `null`.                                       |
+| `undefined` | `Undefined` | This data type is denoted by the keyword `undefined`. It also represents the absence of a value though it has a different use than `null`.    |
+| `symbol` | `Symbol`    | A newer feature to the language, symbols are unique identifiers, useful in more complex coding. No need to worry about these for now.         |
+
+
+## Objects
+Collection of related Data.    
```

Comment:
```suggestion
## Objects

[Objects](https://www.codecademy.com/resources/docs/javascript/objects) are a little more complex since they are collections of related data. 
```